Cargo.toml

```diff
@@ -50,8 +52,11 @@ percent-encoding = "2.0"
 pgp = { version = "0.7.0", default-features = false }
 pretty_env_logger = { version = "0.4.0", optional = true }
 quick-xml = "0.18.1"
+r2d2 = "0.8.5"
+r2d2_sqlite = "0.17.0"
 rand = "0.7.0"
 regex = "1.1.6"
+rusqlite = { version = "0.24", features = ["bundled"] }
```

Comment:
Is there a way to turn the bundled feature off for a distribution package? I guess we'll have to patch `Cargo.toml`?

Reply:
`deltachat-core-rust` is a Rust package too and has its own features. We can change the `[features]` section in `Cargo.toml` to:
```
default = ["bundled-sqlite"]
internals = []
repl = ["internals", "rustyline", "log", "pretty_env_logger", "ansi_term", "dirs"]
vendored = ["async-native-tls/vendored", "async-smtp/native-tls-vendored"]
nightly = ["pgp/nightly"]
bundled-sqlite = ["rusqlite/bundled"]
```

And then you can build with `--no-default-features` if you don't want bundled SQLite. I don't mind hardcoding `--no-default-features` to `CMakeLists.txt` in this case.

Reply:
That would be nice! Hardcoding `--no-default-features` in `CMakeLists.txt` is up to you, I can also change that in Nixpkgs.

Reply:
I am going to do this with `rusqlite` update then.

Since you are packaging tagged release anyway, for now you can just patch `Cargo.toml`.